bin/mail/2016-03-21-teacher-recruitment/generate-ops-workshop-attendees

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'cdo/geocoder'
+
+# Teachers who attended Ops workshops, minus NYC
+
+teacher_query = <<eos
+  SELECT DISTINCT users.email, users.name, users.current_sign_in_ip FROM users
+    JOIN workshop_attendance ON (users.id = workshop_attendance.teacher_id)
+    JOIN segments ON (segments.id = workshop_attendance.segment_id)
+    JOIN workshops ON (workshops.id = segments.workshop_id)
+    WHERE workshops.location NOT LIKE '%NY%'
```

Comment:
Is this to broad? IIRC, that syntax is case-insensitive, meaning it would also rule out "Albany".

Additionally, if we are trying to exclude New York City (per comment), any reason not to do NOT LIKE '%NYC%' (if that string appears for NYC users)?